tests/e2e-openshift/monitoring/04-verify-traces.yaml

```diff
@@ -1,3 +1,28 @@
+# Add the clusterrole and rolebinding required for fetching metrics from Thanos querier. Refer https://issues.redhat.com/browse/MON-3379
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kuttl-cluster-monitoring-metrics-api
+rules:
+- apiGroups: ["monitoring.coreos.com"]
+  resources: ["prometheuses/api"]
+  verbs: ["get", "list", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kuttl-cluster-monitoring-metrics-api
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kuttl-cluster-monitoring-metrics-api
+subjects:
+- kind: ServiceAccount
+  name: prometheus-user-workload
```

Comment:
The tempo operator does not own/manage the `prometheus-user-workload` service account, is it really the responsibility of the tempo operator (tests) to grant this service account additional permissions?

Shouldn't this permission be granted in the CMO operator?

Reply:
> The ClusterRoleBinding step is not a Tempo requirement and is used only by the test case to check the metrics using the check_metrics.sh script.

Ah! Thanks for the explanation. That was the part I didn't realize, that check_metrics.sh is impersonating the `prometheus-user-workload` service account.
Could you add this as a comment above the ClusterRoleBinding?

Reply:
Added the comment. :)

Reply:
Thank you!